joint_state_controller/doc/userdoc.rst

```diff
@@ -3,15 +3,10 @@
 joint_state_controller
 ----------------------
 
-The controller reads all state interfaces and reports them on ``/joint_states`` and ``/dynamic_joint_states``.
+The ``joint_state_controller`` has been renamed to ``joint_state_broadcaster``
+in order to differentiate between true controllers and simple data broadcasters.
 
-Commands
-^^^^^^^^
-
-This is not a real controller, therefore is takes no commands.
-
-Hardware interface type
-^^^^^^^^^^^^^^^^^^^^^^^
-
-All available *joint state interfaces* are used by this controller.
+This package will be removed in ROS H-Turtle. Instead, use package
```

Comment:
Not sure if this ("H-Turle") is correct - Let me know and I can adjust the deprecation notice if required

Reply:
I thinks this is OK. H-Turtle is next LTS and there we want to get rid of this :)